src/annotation.js

```diff
@@ -15,11 +15,45 @@ var annotationId = 0;
 var annotationState = {
   create: 'create',
   done: 'done',
+  highlight: 'highlight',
   edit: 'edit'
 };
 
 var annotationActionOwner = 'annotationAction';
 
+var defaultEditHandleStyle = {
+  fill: true,
+  fillColor: function (d) {
+    return d.selected ? {r: 0, g: 1, b: 1} : {r: 0.3, g: 0.3, b: 0.3};
+  },
+  fillOpacity: function (d) {
+    return d.selected ? 0.5 : 0.25;
+  },
+  radius: function (d) {
+    return d.type === 'edge' || d.type === 'rotate' ? 8 : 10;
+  },
+  scaled: false,
+  stroke: true,
+  strokeColor: {r: 0, g: 0, b: 1},
```

Comment:
I would probably make the default color to orange. What you think @matthewma7 ?

Reply:
@aashish24 The current color works fine for me, because I used some yellow-orange color in the other project.